documentation

```diff
@@ -1,51 +0,0 @@
-Requirements:
-
-database - name, age, number of people in household, address, post code
-reatlational database needed for this
-table of postcode ----- which have addressess ----- each addrss has the number of people livign there and the owner --- the number of poeple is connected with the names and age of the poeple livign there
-
-
-
-
-
-
-
-functianility:
```

Comment:
the three requests look good, would you want to make any others such as post, patch/put, delete